sse2neon.h

```diff
@@ -114,6 +114,11 @@
 /* If using MSVC */
 #ifdef _MSC_VER
 #include <intrin.h>
+
+#if (defined(_M_AMD64) || defined(__x86_64__)) || \
+    (defined(_M_ARM) || defined(__arm__))
+#define SSE2NEON_MSVC_HAS_BITSCAN64
```

Comment:
Shorten it as `SSE2NEON_HAS_BITSCAN64` which will not conflict with other macros or configuration entries.